src/test/java/com/spartasystems/holdmail/EntityPersistenceTest.java

```diff
@@ -116,6 +116,35 @@ public void shouldSaveRecipients() throws Exception {
 
     }
 
+    /* Verify fix for https://github.com/SpartaSystems/holdmail/issues/8
+     *
+     * This test's value is limited to ensuring that the integration suite's
+     * DB of choice isn't hit by this defect (currently h2 which wasn't affected anyway..)
+     *
+     * TxType.NEVER ensures that the save()/load() here actually hits
```

Comment:
http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/test/context/transaction/TransactionalTestExecutionListener.html 
To be honest, it was trial and error that I discovered that it was honored at a test level :)